db/models/task.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { STRING } = require('sequelize');
+
+module.exports = db =>
+  db.define('tasks', {
+    description: STRING,
+    categories: STRING,
+    creator: STRING,
+    assignee: STRING
```

Comment:
Creator and Assignee need to come from two different associations to User:

```Task.belongsTo(User, {as: 'creator'})```
```Task.belongsTo(User, {as: 'assignee'})```

This adds a ```creatorId``` and ```assigneeId``` field to Task, and allows for the methods task.setCreator() and task.setAssignee()